tools/installpsh-osx.sh

```diff
@@ -127,10 +127,22 @@ fi
 
 # Suppress output, it's very noisy on travis-ci
 echo "Refreshing Homebrew cache..."
-if ! brew update > /dev/null; then
-    echo "ERROR: Refreshing Homebrew cache failed..." >&2
-    exit 2
-fi
+for count in {1..2}; do
+    # Try the update twice if the first time fails
+    brew update > /dev/null && break
+
+    # If the update fails again after increasing the Git buffer size, exit with error.
```

Comment:
caching the brew cache in travis-ci might help:
```
cache:
  directories:
    - $HOME/Library/Caches/Homebrew
```

Reply:
Thanks Travis! I will add this.